api/src/main/java/org/openmrs/module/legacyui/LegacyUIModule.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+/**
+ * 
+ */
+package org.openmrs.module.legacyui;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+//import org.openmrs.Patient;
+// import org.openmrs.propertyeditor.PatientEditor;
```

Comment:
Had see this https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode?

Reply:
I'll definitely have a look at that